core/liblwm2m.h

```diff
@@ -497,13 +498,22 @@ typedef struct _lwm2m_observed_
     lwm2m_watcher_t * watcherList;
 } lwm2m_observed_t;
 
+typedef enum {
+    NOT_BOOTSTRAPPED = 0,
+    BOOTSTRAP_REQUESTED,
+    BOOTSTRAP_CLIENT_HOLD_OFF,
+    BOOTSTRAP_PENDING,
+    BOOTSTRAP_FINISHED,
+    BOOTSTRAP_FAILED,
+    BOOTSTRAPPED
```

Comment:
Also more "succesfull BOOTSTRAPED since last reboot". In my opinion, with a potential sequence of bootstrap servers, this may be also valid, if the bootstrap server was exchanged even without a LWM2M Server.